tests/meson.build

```diff
@@ -0,0 +1,12 @@
+gtest = dependency('gtest_main')
```

Comment:
While just `apt-get install`ing on CI is fine for this, I do wonder how dealing with the dep in a cbuildrt environment is supposed to happen.

Reply:
it isn't, we don't have a test stage in xbstrap

when we do, as a host tool